config/uc-r2.yaml

```
@@ -0,0 +1,90 @@
+image:
+  mirrorbits:
+    tag: v0.1.2
+    pullPolicy: IfNotPresent
+  files:
+    # repo & '@sha256' suffix here as we're retrieving a docker digest from updatecli
+    repository: httpd@sha256
+    tag: f70876d78442771406d7245b8d3425e8b0a86891c79811af94fb2e12af0fadeb
+    pullPolicy: IfNotPresent
+
+ingress:
+  enabled: true
+  className: public-nginx
+  annotations:
+    "cert-manager.io/cluster-issuer": "letsencrypt-prod"
+    "nginx.ingress.kubernetes.io/ssl-redirect": "true"
+  hosts:
+    - host: test-hlm-uc-r2
```

```suggestion
    - host: azure.updates.jenkins.io
```

WDYT about using a valid hostname to allow testing the service as soon as it is installed?

---

That was the plan, I added the "test-hlm-uc-r2" CNAME record manually but forgot the `.jenkins.io` here.

(Record manually replaced by "azure.updates")